packages/backend/discovery/pNetwork/discovered.json

```diff
@@ -1,8 +1,30 @@
 {
   "name": "pNetwork",
-  "blockNumber": 16440359,
-  "configHash": "0x9120d358b7473b75ff02b669693c6a9c2e090ee92598c6b542b35db6b5ec8092",
+  "blockNumber": 16720297,
+  "configHash": "0xde648f9842593952677381089354cedd2a969bba27b683a8ba8656820297ddbf",
   "contracts": [
+    {
+      "name": "Erc20Vault",
+      "address": "0x112334f50Cb6efcff4e35Ae51A022dBE41a48135",
+      "code": "https://etherscan.deth.net/address/0x112334f50Cb6efcff4e35Ae51A022dBE41a48135",
+      "upgradeability": {
+        "type": "immutable"
+      },
+      "values": {
+        "PNETWORK": "0xDffE7AC6B538B4A7Fd81c98C5fba0415d63fB132"
+      }
```

Comment:
Why did the order of those contracts change? 🤔

Reply:
I think this is the artifact from the past, the order looks correct, as expected from config